shell/src/main/java/alluxio/cli/ConfigurationDocGenerator.java

```diff
@@ -164,6 +164,10 @@ static void writeYMLFile(Collection<? extends PropertyKey> defaultKeys, String f
         String description = iteratorPK.getDescription().replace("'", "''");
 
         // Write property key and default value to yml files
+        if (iteratorPK.isIgnoredSiteProperty()) {
+          description = "(Alluxio ignores the value you specify for this property "
```

Comment:
Parentheses imply the message is non-essential.

Maybe something like this? 

`description + "Note: This property must be specified as a JVM property; it is not accepted in alluxio-site.properties"`

Reply:
Done.